Integrations/integration-cyberark_AIM.yml

```diff
@@ -304,11 +304,14 @@ script:
       description: Account device type
     description: Query CyberArk AIM
   - name: fetch-credentials
+    description: "Fetches credentials from cyberark vault"
     arguments:
     - name: identifier
+      description: "Identifier to fetch single credentials object by"
   - name: reset-credentials
     arguments:
     - name: immediateChangeByCPM
+      description: "-"
```

Comment:
@Bars92 can you please comment here the description of this argument? `immediateChangeByCPM`